### PR TITLE
t-rdfToFiles: Graph exporting rule condition

### LINKED ISSUE #349: Empty output from e-filesDownload generates non-empty output if passed through t-filesToRdf and t-rdfToFiles
If I use e-filesDownload to get a non-existent file (this raises warning), then pass it to t-filesToRdf and back to t-rdfToFiles it creates non-empty output, such as the following:

```

<http://unifiedviews.eu/resource/internal/dataunit/exec/732/dpu/948/du/0> <http://unifiedviews.eu/ontology/internal/data/read> <http://unifiedviews.eu/resource/internal/dataunit/exec/732/dpu/948/du/0> ;
	<http://unifiedviews.eu/ontology/internal/data/write> <http://unifiedviews.eu/resource/internal/dataunit/exec/732/dpu/948/du/0> ;
	<http://unifiedviews.eu/ontology/internal/data/counter> "0"^^<http://www.w3.org/2001/XMLSchema#int> .

<http://unifiedviews.eu/resource/internal/dataunit/exec/732/dpu/951/du/0> <http://unifiedviews.eu/ontology/internal/data/read> <http://unifiedviews.eu/resource/internal/dataunit/exec/732/dpu/948/du/0> , <http://unifiedviews.eu/resource/internal/dataunit/exec/732/dpu/951/du/0> ;
	<http://unifiedviews.eu/ontology/internal/data/write> <http://unifiedviews.eu/resource/internal/dataunit/exec/732/dpu/951/du/0> ;
	<http://unifiedviews.eu/ontology/internal/data/counter> "0"^^<http://www.w3.org/2001/XMLSchema#int> .

<http://unifiedviews.eu/resource/internal/dataunit/exec/732/dpu/951/du/1> <http://unifiedviews.eu/ontology/internal/data/read> <http://unifiedviews.eu/resource/internal/dataunit/exec/732/dpu/951/du/1> ;
	<http://unifiedviews.eu/ontology/internal/data/write> <http://unifiedviews.eu/resource/internal/dataunit/exec/732/dpu/951/du/1> ;
	<http://unifiedviews.eu/ontology/internal/data/counter> "0"^^<http://www.w3.org/2001/XMLSchema#int> .

<http://unifiedviews.eu/resource/internal/dataunit/exec/732/dpu/950/du/1/entry/1> <http://unifiedviews.eu/DataUnit/MetadataDataUnit/symbolicName> "still_not_there.ttl" ;
	<http://unifiedviews.eu/DataUnit/MetadataDataUnit/FilesDataUnit/fileURI> "file:/Users/mynarzjindrich/Dev/UnifiedViews/app/Core/backend/working/exec_732/storage/dpu_950/1/still_not_2252596351209794943" ;
	<http://unifiedviews.eu/VirtualPathHelper/virtualPath> "still_not_there.ttl" .
```

This looks a lot like metadata, so it seems that the DPUs mix up data with metadata.

Context: All used DPUs were compiled from the latest develop commit (33ceb3dbcf4bf0499e50783c5c0b41b6ecbc36ff).

### PR DESCRIPTION
**Pull request by mbl-eea-sk**
Now DPU t-rdfToFiles produces content only if list of files received from e-filesDownload is not empty.
In t-filesToRdf the creation of update query was refactored from Strings concatenation to a method.
Resolves #349

P.S.
When importing an existing file this pipeline failed even with all DPUs involved taken from develop branch. Not sure if this is correct behavior or the input should be configured in another way.

### CODE REVIEW

**Comment by tomas-knap** [comment]
Waiting for feedback from @jindrichmynarz  to https://github.com/UnifiedViews/Plugins/issues/349.

**Comment by tomas-knap** [comment]
@mbl-eea-sk: This is not clear to me: "When importing an existing file this pipeline failed even with all DPUs involved taken from develop branch. Not sure if this is correct behavior or the input should be configured in another way. " Which pipeline do you mean by "this pipeline"?

**Comment by mbl-eea-sk** [comment]
Pipeline: e-filesDownload -> t-filesToRdf -> t-rdfToFiles -> l-filesUpload
Seems that this issue is declared in #350

**Comment by tomas-knap** [comment]
Should be finalized based in inputs in #349

File: t-rdfToFiles/CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,10 @@
 T-RdfToFiles
 ----------
 
+2.2.0
```
Comment (skrchnavy) [inline t-rdfToFiles/CHANGELOG.md]: this is not big change, enough shall be to upgrade patch version (2.1.5-SNAPSHOT). Also must be sync with pom.xml

**Comment by tomas-knap** [comment]
@mbl-eea-sk  If you look at changed files in the pull request, there is also t-filesToRdf/pom.xml. Probably you wanted to change t-rdfToFiles/pom.xml, right?

**Comment by mbl-eea-sk** [comment]
Sorry, I forgot to switch files in my editor. Adjusted.

**Pull request merged**